db/migrate/20160831054910_old_night_unit_to_custom_unit.rb

```diff
@@ -0,0 +1,270 @@
+      "fr",
+      "fr-CA",
+      "it",
+      "ja",
+      "nb",
+      "nl",
+      "pt-BR",
+      "ru",
+      "sv",
+      "tr-TR",
+      "zh",
+      "pt-PT",
+      "ca",
+      "en-NZ",
+      "et",
+      "hr",
+      "id",
+      "is",
+      "km-KH",
+      "ms-MY",
+      "pl",
+      "ro",
+      "sw",
+      "vi",
+      "hu",
+      "cs",
+      "th-TH",
+      "bg",
+      "mn",
+      "zh-TW",
+      "zh-HK",
+      "ka",
+      "sl",
+      "sk-SK"
+    ])
+
+  end
+
+  def night_unit_translations
+    {
+      "da-DK"=>"nat",
+      "de"=>"Nacht",
+      "el"=>"night",
+      "en"=>"night",
+      "en-AU"=>"night",
+      "en-GB"=>"night",
+      "es-ES"=>"noche",
+      "es"=>"noche",
+      "fi"=>"yö",
+      "fr"=>"nuit",
+      "fr-CA"=>"nuit",
+      "it"=>"notte",
+      "ja"=>"夜",
+      "nb"=>"natt",
+      "nl"=>"nacht",
+      "pt-BR"=>"noite",
+      "ru"=>"ночь",
+      "sv"=>"natt",
+      "tr-TR"=>"gecelik",
+      "zh"=>"晚",
+      "pt-PT"=>"noite",
+      "ca"=>"night",
+      "en-NZ"=>"night",
+      "et"=>"night",
+      "hr"=>"night",
+      "id"=>"night",
+      "is"=>"night",
+      "km-KH"=>"night",
+      "ms-MY"=>"night",
+      "pl"=>"night",
+      "ro"=>"night",
+      "sw"=>"night",
+      "vi"=>"night",
+      "hu"=>"éjszaka",
+      "cs"=>"noc",
+      "th-TH"=>"night",
+      "bg"=>"night",
+      "mn"=>"night",
+      "zh-TW"=>"晚",
+      "zh-HK"=>"晚",
+      "ka"=>"ღამე",
+      "sl"=>"night",
+      "sk-SK"=>"noc"
+    }
+  end
+
+  def night_selector_translations
+    {
+      "da-DK"=>"Antal nætter:",
+      "de"=>"Anzahl Nächte:",
+      "el"=>"Number of nights:",
+      "en"=>"Number of nights:",
+      "en-AU"=>"Number of nights:",
+      "en-GB"=>"Number of nights:",
+      "es-ES"=>"Número de noches:",
+      "es"=>"Número de noches:",
+      "fi"=>"Öiden määrä:",
+      "fr"=>"Nombre de nuits : ",
+      "fr-CA"=>"Nombre de nuits : ",
+      "it"=>"Numero di notti:",
+      "ja"=>"宿泊日数：",
+      "nb"=>"Antall netter:",
+      "nl"=>"Number of nights:",
+      "pt-BR"=>"Número de noites:",
+      "ru"=>"Количество недель:",
+      "sv"=>"Antal nätter:",
+      "tr-TR"=>"Number of nights:",
+      "zh"=>"夜晚数量:",
+      "pt-PT"=>"Número de noites:",
+      "ca"=>"Number of nights:",
+      "en-NZ"=>"Number of nights:",
+      "et"=>"Number of nights:",
+      "hr"=>"Number of nights:",
+      "id"=>"Number of nights:",
+      "is"=>"Number of nights:",
+      "km-KH"=>"Number of nights:",
+      "ms-MY"=>"Number of nights:",
+      "pl"=>"Number of nights:",
+      "ro"=>"Number of nights:",
+      "sw"=>"Number of nights:",
+      "vi"=>"Number of nights:",
+      "hu"=>"Éjszakák száma:",
+      "cs"=>"Počet nocí:",
+      "th-TH"=>"Number of nights:",
+      "bg"=>"Number of nights:",
+      "mn"=>"Number of nights:",
+      "zh-TW"=>"夜晚数量:",
+      "zh-HK"=>"晚：",
+      "ka"=>"ღამეების რაოდენობა:",
+      "sl"=>"Number of nights:",
+      "sk-SK"=>"Počet nocí:"
+    }
+  end
+end
```

Comment:
wow, this migration script underlines the reasons why a) custom units and b) community_translations are not easy to handle like they are today.